performance-storage-service/pss_project/api/views/git_events.py

```diff
@@ -34,75 +39,51 @@ def create(self, request):
             logger.debug('Authenticated with Github repo')
 
             if(not repo_client.is_valid_installation_id(payload.get('installation', {}).get('id'))):
-                return Response({"message": "This app only works with the NoisePage repo"}, status=HTTP_400_BAD_REQUEST)
+                return Response({"message": "This app only works with the NoisePage repo"},
+                                status=HTTP_400_BAD_REQUEST)
 
             if event == 'pull_request':
                 handle_pull_request_event(repo_client, payload)
             if event == 'status':
                 handle_status_event(repo_client, payload)
-                
+
         except Exception as err:
-            return Response({"message": err.message if hasattr(err,'message') else err}, status=HTTP_500_INTERNAL_SERVER_ERROR)
-        
+            return Response({"message": err.message if hasattr(err, 'message') else err},
+                            status=HTTP_500_INTERNAL_SERVER_ERROR)
+
         return Response(status=HTTP_200_OK)
 
+
 def is_valid_github_webhook_hash(hash_header, req_body):
     """ Check that the has passed with the request is valid based on the
     webhook secret and the request body """
-    alg, req_hash = hash_header.split('=',1)
-    valid_hash = hmac.new(str.encode(WEBHOOK_SECRET),req_body, alg)
+    alg, req_hash = hash_header.split('=', 1)
+    valid_hash = hmac.new(str.encode(WEBHOOK_SECRET), req_body, alg)
     return hmac.compare_digest(req_hash, valid_hash.hexdigest())
 
+
 def handle_pull_request_event(repo_client, payload):
-    """ When a pull request event is detected create a new check run for the 
+    """ When a pull request event is detected create a new check run for the
     performance cop"""
     # TODO: Not sure if we want some logic for labels
-    #TODO: This if statement is temporary so we can test in the NoisePage repo without affecting everyone
-    if payload['pull_request'].get('user',{}).get('login') == 'bialesdaniel': 
+    # TODO: This if statement is temporary so we can test in the NoisePage repo without affecting everyone
+    if payload['pull_request'].get('user', {}).get('login') == 'bialesdaniel':
         commit_sha = payload['pull_request'].get('head', {}).get('sha')
         if should_initialize_check_run(payload.get('action')):
             logger.debug('pull request reinitialized check')
             initialize_check_run(repo_client, commit_sha)
         if payload.get('action') == 'closed':
-            logger.debug('pull request was closed')
-            branch = payload['pull_request'].get('head',{}).get('ref')
+            logger.debug('pull request was closed ')
+            branch = payload['pull_request'].get('head', {}).get('ref')
             cleanup_check_run(branch)
```

Comment:
Does it only check your github login or is it just a testing data here

Reply:
Yah this is temporary just for testing